services/director-v2/src/simcore_service_director_v2/utils/client_decorators.py

```diff
@@ -71,17 +77,25 @@ async def wrapper_func(*args, **kwargs) -> httpx.Response:
             if httpx.codes.is_client_error(resp.status_code):
                 raise HTTPException(resp.status_code, detail=resp.reason_phrase)
             if httpx.codes.is_server_error(resp.status_code):  # i.e. 5XX error
-                logger.error(
-                    "%s service error:\n|Request|\n%s\n%s\n%s\n|Response|\n%s\n%s\n%s",
-                    service_name,
-                    resp.request,
-                    _format_headers(resp.request.headers),
-                    resp.request.content.decode(),
-                    resp,
-                    _format_headers(resp.headers),
-                    resp.text,
-                )
-                raise HTTPException(status.HTTP_503_SERVICE_UNAVAILABLE)
+                # If server error comes from legacy service retrieve path (/x/UUID_4/retrieve) we log as info
+                if LEGACY_SERVICE_RETRIEVE_PATH_PATTERN.match(resp.url.path):
+                    logger.info(
+                        "%s legacy service does not implement %s",
```

Comment:
I am not convinced by this solution: If the service does not implement this entrypoint, typically it will respond with a https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404  but this is a client error (i.e. 4XX) and not a server error (i.e. 5XX) !  

Here instead, any kind of server error (i.e. 5XX) from a ``retreive`` call to a legacy is completely bypassed instead!

Reply:
So what we found out by actually looking into this error is the following: the service (which is called directly by director-v2) is returning an 500 error and produces some logs.
There are two issues now:
1. how we deal with this reply and what we report to the frontend
2. how to deal with the actual service that is raising the error

Reply:
@sanderegg @pcrespov some feedback on the above would be welcomed.

Reply:
my 2 cents here is that the call is made too soon. and the server is not up.

Reply:
Yes, it seems like it, but it is interesting that it is happening just for these two services. Neither less if you don't object I would close this PR (as the change to ignore the 500 on /retrieve endpoint is probably not the best idea (as the 500 is the correct behavior) and we can discuss whether to do something about it.